features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Monetization/BusinessPlans.jsx

```diff
@@ -80,126 +105,102 @@ class BusinessPlans extends Component {
+                    })}
+
+                </Table>
             </Grid>
         ));
         return (
-            <Grid container className={classes.root}>
-                <Grid className={classes.grid} spacing={2}>
-                    <Grid>
-                        <Typography variant='subtitle' gutterBottom>
-                            <FormattedMessage
-                                id='Apis.Details.Monetization.BusinessPlans.commercial.policies'
-                                defaultMessage='Commercial Policies'
-                            />
-                        </Typography>
+            <Grid container spacing={1}>
+                <Grid item xs={12}>
+                    <Typography variant='h6'>
+                        <FormattedMessage
+                            id='Apis.Details.Monetization.BusinessPlans.commercial.policies'
+                            defaultMessage='Commercial Policies'
+                        />
+                    </Typography>
+                </Grid>
+                <Grid item xs={12}>
+                    <Grid container direction='row' spacing={3}>
+                        {policiesList}
                     </Grid>
+                </Grid>
+                <Grid item xs={12}>
                     {
                         (policies.length > 0) ? (
-                            <Grid>
-                                <Typography>
-                                    <FormattedMessage
-                                        id='Apis.Details.Monetization.BusinessPlans.unchecked.policies'
-                                        defaultMessage='Unchecked polices are not monetized, click `Save` to monetize'
-                                    />
-                                </Typography>
-                            </Grid>
+                            <Banner
+                                disableActions
+                                dense
+                                paperProps={{ elevation: 1 }}
+                                type='info'
+                                message='Click Save to monetize all unmonetized policies'
+                            />
                         ) : (
-                            <Grid>
-                                <Typography>
-                                    <FormattedMessage
-                                        id='Apis.Details.Monetization.BusinessPlans.no.commercial.policies.to.monetize'
-                                        defaultMessage='No commercial policies to monetize'
-                                    />
-                                </Typography>
-                            </Grid>
+                            <Banner
+                                disableActions
+                                dense
+                                paperProps={{ elevation: 1 }}
+                                type='info'
+                                message='No commercial policies to monetize'
```

Comment:
Need i18n

Reply:
The support fot i18n is not yet available in above Banner component at [Banner](https://github.com/wso2/carbon-apimgt/blob/master/features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Shared/Banner.jsx#L65)